src/android/Player.java

```diff
@@ -92,6 +92,10 @@ public void onDismiss(DialogInterface dialog) {
         public boolean onKey(DialogInterface dialog, int keyCode, KeyEvent event) {
             JSONObject payload = Payload.keyEvent(event);
             new CallbackResponse(Player.this.callbackContext).send(PluginResult.Status.OK, payload, true);
+            
+            //Added method to handle remote key events to show controls on any key event
+            exoView.showController();
```

Comment:
This will only show ExoPlayer's controller on the bottom. We also have to support info header on the top which needs to be synchronized with the bottom controlller.

Can you take a look at onTouchListener and see if you can make it work in similar fashion?

Reply:
@mkalyon @FGMGIT is this still an issue? What's the proper way to fix this issue?